fluentui_core/src/main/res/values/attrs.xml

```diff
@@ -34,4 +34,10 @@
     <!--Transparent Foregrounds-->
     <attr name="fluentuiForegroundOnPrimary80Color" format="reference|color"/>
     <attr name="fluentuiForegroundOnPrimary70Color" format="reference|color"/>
+
+    <!--fluent_others Start-->
+
+    <!--ActionBarLayout-->
+    <attr name="fluentui_viewPager" format="reference"/>
```

Comment:
NIT : We can add a new line here

Please check other PR's. also

Reply:
Sure, I will take care while merging it to develop branch.